apps/docs/docs/modules/data_loaders/discord.mdx

```diff
@@ -20,7 +20,7 @@ Copy the URL in your browser and select the server you want your bot to join.
 #### DiscordReader()
 
 - `discordToken?`: The Discord bot token.
-- `makeRequest?`: Optionally provide a custom request function for edge environments, e.g. `fetch`. See discord.js for more info.
+- `requestHandler?`: Optionally provide a custom request function for edge environments, e.g. `fetch`. See discord.js for more info.
```

Comment:
why this change in the llamaparse PR?

Reply:
I just noticed it and it's just a change in the docs, I mistakenly called it `makeRequest` in my previous PR, which is not the actual param name.

Reply:
@KindOfAScam Great catch! Thanks a lot. To avoid confusion, better send a separate PR next time